src/app/digital/models/ioobjects/gates/XORGate.ts

```diff
@@ -24,15 +30,22 @@ export class XORGate extends Gate {
         super.activate(on);
     }
 
+    /***
+     * Returns a vector which helps position the component's inputs when more inputs are added.
+     */
     // @Override
     public getOffset(): Vector {
         return super.getOffset().add(0, GetQuadraticOffset(this.numInputs()));
     }
-
+    /***
+     * Returns the name of the gate
+     */
```

Comment:
For instance here, I might write:

/**
\* \@returns string representing the name of the gate
\* /
(if all a function does is return a value, you can probably skip including a description outside of the @returns tag)